lib/frontegg/hierarchy.rb

```diff
@@ -0,0 +1,10 @@
+module Frontegg
+  class Hierarchy < Resource
```

Comment:
not sure about this, the developer doesn't need to know the weirdness of frontegg API decisions, I'm ok with this living in tenant class with the method defining its own path